package.json

```diff
@@ -13,7 +13,7 @@
     "clean": "rm -rf dist/",
     "commit": "git-cz",
     "commit:retry": "npm run commit -- --retry",
-    "test": "jest",
+    "test": "jest --runInBand",
```

Comment:
nope, just for the ci =), so either add a new script or run `npm run test -- --runInBand` in circle ci